staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2840,6 +2845,20 @@ const (
 	RestartPolicyNever     RestartPolicy = "Never"
 )
 
+// ContainerRestartPolicy is the restart policy for a single container.
```

Comment:
"""
ContainerRestartPolicy is the restart policy for a single container.  The only supported value is "Always".
"""